documentation/decisions/adr/2023-12-18-measurement-dashboard-architecture.md

```diff
@@ -0,0 +1,149 @@
+  - Hardest to maintain of all of the options considered.
+  - Does not easily support adhoc reports or dashboards without using a separate tool.
+
+### Custom dashboard app
+
+This option involves building a custom dashboard app using an existing framework, such as [Dash][plotly-dash] (python) or [R Shiny][r-shiny]. This custom built application would be hosted as its own application and be slightly more integrated than a separate analytics API + dashboard UI, but it would be more customizable than using an open source or proprietary dashboard solution.
+
+> [!TIP]
+> **Bottom line:** This option would be best if we want to retain control over the look and feel of the dashboard and manage the backend and frontend of the dashboard together, but can dedicate more engineering resources to building and hosting it and don't need to expose analytics data via API.
+
+- **Pros**
+  - Provides more fine-grained control over the look and feel of the dashboard application, on par with option 1.
+  - Allows a single engineer familiar with Python (or R if we use Shiny) to manage both the backend (e.g. analytics) and frontend (visualization and design of the dashboard).
+  - Faster to implement new charts and dashboards than an analytics API and separate frontend.
+  - Enables us to combine multiple dashboards into a single application.
+  - Allows open source contributors to host local versions of our dashboards.
+- **Cons**
+  - Building or modifying dashboards is still quite technical, not something a business analyst could do on their own.
+  - Harder to implement new charts or dashboards than SaaS or open source dashboard solution.
+  - Does not easily support adhoc reports or dashboards in the same tool.
+  - Does not easily support exposing analytics data to S2S users via API.
+
+### Open source dashboard solution
+
+This option involves selecting and hosting an open source dashboard solution, such as [Metabase][metabase] or [Redash][redash]. This solution would most likely be self-hosted and connect directly to our data warehouse, and enable business analysts to build adhoc reporting and dashboards with SQL and a drag-and-drop interface. Individual dashboards can then be configured for broader publication to external stakeholders.
+
+> [!TIP]
+> **Bottom line:** This option would be best if we want to adopt an open source solution that enables business analysts to build and host dashboards with minimal support from engineers, but are willing to compromise on the amount of control we have over the look and feel of those dashboards and don't need to expose analytics data via API.
+
+- **Pros**
+  - Enables a business analyst or engineer with basic SQL experience to build and manage dashboards.
+  - Faster and easier to implement new charts and dashboards than options 1 or 2.
+  - Supports adhoc reports and dashboards in the same tool.
+  - Allows open source contributors to host local versions of our dashboards (depending on the tool we choose).
```

Comment:
Is it free? That's also a pro. 

And we should add, fits open source development model of the project (or something like that).

Reply:
Adds these notes in the following commits:
- [fix: Clarifies pros + cons for dashboard solution options](https://github.com/HHS/simpler-grants-gov/pull/936/commits/8ab6dcf050dc4697eec405468ce870e9d9b8eb28)
- [feat: Adds note about open source values](https://github.com/HHS/simpler-grants-gov/pull/936/commits/769e1a042855de51c8ac8f240a3b6da3ec80dc87)